### PR TITLE
Add NTP port.

### PR DESCRIPTION
**Pull request by haydenroche5**
Should be tested with https://github.com/wolfSSL/wolfssl/pull/4165.

### CODE REVIEW

File: ntp/4.2.8p15/ntp-4.2.8p15.patch [hunk 1]
```diff
@@ -0,0 +1,533 @@
...
+  */
+@@ -22,28 +25,28 @@
+ #  include <md5.h>
+ # else
+ #  include "isc/md5.h"
+-   typedef isc_md5_t		MD5_CTX;
+-#  define MD5Init(c)		isc_md5_init(c)
+-#  define MD5Update(c, p, s)	isc_md5_update(c, p, s)
+-#  define MD5Final(d, c)	isc_md5_final((c), (d))	/* swapped */
++   typedef isc_md5_t    MD5_CTX;
```
Comment (cconlon) [inline ntp/4.2.8p15/ntp-4.2.8p15.patch:186]: This block of changes seems to be spacing only.  If we put these back, we might benefit from a shorter patch file?

File: ntp/4.2.8p15/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Build Instructions
+
+### Build wolfSSL
++ Configure wolfSSL with `./configure --enable-certgen --enable-dsa --enable-cmac --enable-opensslextra --enable-opensslall --enable-keygen`. Add `--enable-debug` if you want to enable the debug version of wolfSSL.
++ Compile with `make`.
++ Install wolfSSL into /usr/local with `sudo make install`.
+
+### Build NTP
++ Download ntp 4.2.8p15 with `curl -O https://www.ntp.org/downloads/ntp-4.2.8p15.tar.gz`.
```
Comment (cconlon) [inline ntp/4.2.8p15/README.md:9]: Appears that ntp.org is down at the moment for me ("Not Found" error when visiting site).  I grabbed this tar.gz from elsewhere and verified these steps work though.  Guessing ntp.org will come back soon.

Comment (haydenroche5) [reply ntp/4.2.8p15/README.md:9]: Looks to be back up now, at least for me.

File: ntp/4.2.8p15/ntp-4.2.8p15.patch (hunk 1 above)
Comment (haydenroche5) [reply ntp/4.2.8p15/ntp-4.2.8p15.patch:186]: Good catch. Will remove.

**Review by cconlon — APPROVED** [review]

**Pull request merged**